website/docs/docs/dbt-cloud-apis/sl-graphql.md

```diff
@@ -413,11 +415,36 @@ order_total  ordered_at
 """
 ```
 
-### Additional Create Query examples 
+### Additional create query examples 
 
-The following section provides query examples for the GraphQL API, such as how to query metrics, dimensions, where filters, and more.
+The following section provides query examples for the GraphQL API, such as how to query metrics, dimensions, where filters, and more:
 
-**Query two metrics grouped by time**
+    - [Query metric alias](#query-metric-alias)
+    - [Query two metrics grouped by time](#query-two-metrics-grouped-by-time)
+    - [Query with a time grain](#query-with-a-time-grain)
+    - [Query two metrics with a categorical dimension](#query-two-metrics-with-a-categorical-dimension)
+    - [Query a categorical dimension on its own](#query-a-categorical-dimension-on-its-own)
+    - [Query with a where filter](#query-with-a-where-filter)
+    - [Query with order](#query-with-order)
+    - [Query with limit](#query-with-limit)
+    - [Query with just compiling SQL](#query-with-just-compiling-sql)
+    - [Querying compile SQL with saved queries](#querying-compile-sql-with-saved-queries)
+    - [Create query with saved queries](#create-query-with-saved-queries)
+
+#### Query metric alias
+
+```graphql
+mutation {
+  createQuery(
+    environmentId: "..."
+    metrics: [{name: "metric_name", alias: "metric_alias"}]
+  ) {
+    ...
+  }
+}
+```
+
+#### Query two metrics grouped by time
```

Comment:
Can we remove this example? It doesn't do what it says (only one metric, groups by both time and categorical dimensions) and it is pretty redundant with "Query two metrics with a categorical dimension".